wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -195,13 +195,14 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
      */
 
     @Throws(Exception::class)
-    suspend fun searchUsernames(text: String, onlyExactUsername: Boolean = false): List<UsernameSearchResult> {
+    suspend fun searchUsernames(text: String, onlyExactUsername: Boolean = false, limit: Int = -1): List<UsernameSearchResult> {
         val userIdString = blockchainIdentity.uniqueIdString
         val userId = blockchainIdentity.uniqueIdentifier
 
         // Names.search does support retrieving 100 names at a time if retrieveAll = false
         //TODO: Maybe add pagination later? Is very unlikely that a user will scroll past 100 search results
-        val nameDocuments = platform.names.search(text, Names.DEFAULT_PARENT_DOMAIN, false)
+        val nameDocuments = platform.names.search(text, Names.DEFAULT_PARENT_DOMAIN,
```

Comment:
This solution relies on https://github.com/dashevo/android-dashpay/pull/30/files which has merge conflicts due to parameter order differences with `Names.search`.  Either the merge conflicts on the previously mentioned `android-dashpay` PR must be resolved or the parameter order must be changed here.

Reply:
Well spotted!
Fixed on https://github.com/dashevo/dash-wallet/pull/566/commits/8fa83414a5a5a464b4055d4efd40d6d350efccbc